--- Crypt-xDBM_File-0.80/test.pl.orig	Tue Aug  1 08:38:11 2000
+++ Crypt-xDBM_File-0.80/test.pl	Fri Aug  2 21:16:26 2002
@@ -55,7 +55,7 @@
 print "key cult = $hash{'cult'}\n";
 untie %hash;
 
-tie %hash, 'Crypt::xDBM_File', 'DES', '01234567', 'SDBM_File', "/tmp/bob.blf", O_RDWR|O_CREAT, 0640;
+tie %hash, 'Crypt::xDBM_File', 'Crypt::DES', '01234567', 'SDBM_File', "/tmp/bob.blf", O_RDWR|O_CREAT, 0640;
 
 $hash{'bob'} = "bob rules!";
 $hash{'of'} = "yes";
